orcestra_book/data_concept.md

```diff
@@ -65,39 +65,37 @@ dataset = get(dataset_list, dataset_identifier)
-* The dataset **must** be available as [`xarray.Dataset`](https://docs.xarray.dev/en/stable/user-guide/data-structures.html#dataset) (it **can** be available in different forms in addition).
-
-
-:::{tip}
-Many formats can be read as an xarray Dataset. This includes e.g. netCDF, zarr, GRIB, CSV, AMES FFI etc..., but it might require supplying an appropriate read routine.
-:::
+The dataset **must** be stored in one the following data formats:
+  * NetCDF
+  * Zarr
+  * CSV
+  * GRIB
+  * AMES FFI
+* It **can** be available in different forms in addition
+* The list of accepted data formats **can** be extended, if the data format is well standardized and readable by several common programming languages
 
 ### 3. datasets are **well-formed** and **analysis-ready**
 
-* You **should** stick to well-known formats (e.g. netCDF / zarr). The chosen format **must** be convertible to `xarray.Dataset`, see above.
 * You **should** follow standard metadata schemes ([CF-Conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html))
 * You **may** designate data processing levels to help users understand the quality of the provided data. If you provide processing levels, the levels **should** follow the [EOSDIS data processing levels](https://www.earthdata.nasa.gov/engage/open-data-services-and-software/data-information-policy/data-levels) scheme.
 * You **should** work with your own **published** datasets.
 * Datasets **should** be reviewed across teams.
 
 ### 4. incremental backups are possible
 
-*(this section is currently in an early stage)*
-
-* storage location has to provide a method to check if something changed
-    * e.g. HTTP ETag, hash of content etc...
-* how to handle updates / changes to a dataset?
-    * ensure that dataset name changes on storage (e.g. version number, hash ...)
-    * list / catalog may point to most recent version
+* You **must** provide a version number for your dataset
+* You *should* add a content identifier for your dataset (specify has algorithm?)
+* The catalog entry **should** point to the most recent version of the dataset
+* The storage location **may** provide a method to check if something changed ( e.g. HTTP ETag, ...)
 
 ### 5. datasets are on a shared, distributed system
 
-*(this section is currently in an early stage)*
+Use a distributed storage protocol to make datasets accessible e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)
 
-use a distributed storage protocol to make datasets accessible
-e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)
-
-*maybe needs more support from computing centers*
+:::{info}
+The implementation of this proposal is likely not ready in time.
+However, further investigation and feasibility studies may be carried out during the campaing.
+:::
```

Comment:
```suggestion
```
I wouldn't be so pessimistic...
There's apparently more support coming up at least from DKRZ and possible HALO-DB will also shift towards this direction. Also we are accessing EUREC4A data this way routinely (e.g. in CI).
We however should examine which way to go. IPFS and ONEDATA would be quite different approaches, also ONEDATA seems to be quite a bit more effort to set up.

Reply:
I am not as optimistic as you, but maybe we shouldn't give up in the concept already :)